File: chroma-league/src/main/java/com/bonepl/chromaleague/hud/animations/AllyCloudDragonKillAnimation.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.rest.eventdata.DragonType;

public class AllyCloudDragonKillAnimation extends StaticBlinkingAnimation {
    public AllyCloudDragonKillAnimation() {
        super(8, DragonType.CLOUD.getColor());
    }
}

File: src/hud/animations/AllyCloudDragonKillAnimation.ts
import {DragonType} from '../../rest/eventdata/DragonType.js';
import {AllyDragonKillBlinkingAnimation} from './AllyDragonKillBlinkingAnimation.js';

export class AllyCloudDragonKillAnimation extends AllyDragonKillBlinkingAnimation {
    constructor() {
        super(DragonType.CLOUD);
    }
}
